CIScripts/BuildAll.ps1

```diff
@@ -19,8 +19,12 @@ Invoke-ContrailCommonActions -ThirdPartyCache $Env:THIRD_PARTY_CACHE_PATH -VSSet
 
 $ReleaseMode = [bool]::Parse($Env:BUILD_IN_RELEASE_MODE)
 
-Invoke-DockerDriverBuild -DriverSrcPath $Env:DRIVER_SRC_PATH -SigntoolPath $Env:SIGNTOOL_PATH -CertPath $Env:CERT_PATH -CertPasswordFilePath $Env:CERT_PASSWORD_FILE_PATH
-Invoke-ExtensionBuild -ThirdPartyCache $Env:THIRD_PARTY_CACHE_PATH -SigntoolPath $Env:SIGNTOOL_PATH -CertPath $Env:CERT_PATH -CertPasswordFilePath $Env:CERT_PASSWORD_FILE_PATH -ReleaseMode $ReleaseMode
-Invoke-AgentBuild -ThirdPartyCache $Env:THIRD_PARTY_CACHE_PATH -SigntoolPath $Env:SIGNTOOL_PATH -CertPath $Env:CERT_PATH -CertPasswordFilePath $Env:CERT_PASSWORD_FILE_PATH -ReleaseMode $ReleaseMode
+New-Item -Path "output" -Name "docker_driver" -ItemType directory
```

Comment:
I think  "New-Item -ItemType directory -Path "output/docker_driver"  looks a bit better.

What do you think about redirecting output of New-Item to /dev/null ?

Reply:
We could also do ForEach($dir in $Dirs) {New-Item ...} - but this is just a thought